run_scripts/run_pipeline.sh

```diff
@@ -32,14 +33,17 @@ echo "Starting run with id '$RUN_ID'"
 ./2_fetch_raw_data.sh "$USER" "$AVF_BUCKET_CREDENTIALS_PATH" "$PIPELINE_CONFIGURATION" "$DATA_ROOT"
 
```

Comment:
I think we should remove the previous outputs folder so that we don't accidentally upload analysis files which are unchanged since last time or include files which were not updated in this run in the backup archive. In both cases, partly because of the inefficiency, and partly because of the potential confusion this may cause later when looking back over the files.

Reply:
yup, updated to : here and in the automated analysis files
`if [ -d "$DATA_ROOT/Outputs" ]
then
    rm -r "$DATA_ROOT/Outputs"
fi

mkdir -p "$DATA_ROOT/Outputs"`